Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -245,23 +256,28 @@ public void centerJunctionTop(GrabberCameraPipeline pipeline) {
      * @param pipeline the RobotCameraPipeline being used by the robot camera
      */
     public void centerConeStack(RobotCameraPipeline pipeline) {
-        double xOffset, width;
-
+        double xOffset, width, strafePower, turnPower;
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
+
+            //This xOffset is different because it is from a different point of view.
```

Comment:
Different from what? I know what you're talking about, but it's not clear if you're just looking at the code. You should explain the point of view for each `xOffset` to be clearer

Reply:
Outdated, the comment was removed.